### PR TITLE
Updated Roadmap to reflect the announcement dates

### PR DESCRIPTION
**Pull request by kevinjpetersen**
Updated Roadmap to reflect announcement dates from BUILD 2019.

### CODE REVIEW

**Comment by arphox** [comment]
_For more info, see article [Introducing .NET 5](https://devblogs.microsoft.com/dotnet/introducing-net-5/)._

**Comment by kevinjpetersen** [comment]
> I am not sure what "GA" means at non-LTS releases in the [introduction article](https://devblogs.microsoft.com/dotnet/introducing-net-5/), but if you think that should not be indicated here, I approve.

It means "General Availability" / Final Release. Maybe it would make sense to put it there, but I think people is going to understand "Final Release" more than "GA/General Availability". Just my opinion tho :) 

Thoughts?

**Comment by karelz** [comment]
Thanks for your change. Let's use terminology from the article to avoid further confusion. We can add full text in parenthesis.
We should also link the article.

File: roadmap.md [hunk 1]
```diff
@@ -19,7 +19,13 @@ For released versions of the product:
 | Milestone                 | Release Date |
 |---------------------------|--------------|
 | .NET Core 2.2.x, 2.1.x, 1.x (servicing) | Approximately every 1-2 months or as needed (see also [releases](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md#net-core-21)) |
-| .NET Core 3.0 | [Preview releases](https://github.com/dotnet/core/blob/master/release-notes/3.0/README.md) <br/> Final version in second half of 2019 - see [Schedule in Preview 3](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-preview-3) (from 2019/3). Ship date will be announced at the Build 2019 conference. <br/> Previous announcements: [original 2018/5](https://blogs.msdn.microsoft.com/dotnet/2018/05/07/net-core-3-and-support-for-windows-desktop-applications/) and [update 2018/10](https://blogs.msdn.microsoft.com/dotnet/2018/10/04/update-on-net-core-3-0-and-net-framework-4-8/). |
+| .NET Core 3.0 | [Preview releases](https://github.com/dotnet/core/blob/master/release-notes/3.0/README.md)<br/>Release Candidate scheduled for July 2019<br/>Final Release scheduled for September 2019 |
```
Comment (karelz) [inline roadmap.md]:
```suggestion
| .NET Core 3.0 | [Preview releases](https://github.com/dotnet/core/blob/master/release-notes/3.0/README.md)<br/>RC (Release Candidate) scheduled for July 2019<br/>GA (General Availability) scheduled for September 2019 |
```

File: roadmap.md [hunk 2]
```diff
@@ -19,7 +19,13 @@ For released versions of the product:
...
+| .NET Core 3.1 LTS | Release scheduled for November 2019 |
+| .NET 5.0 | Release scheduled for November 2020 |
+| .NET 6.0 LTS | Release scheduled for November 2021 |
+| .NET 7.0 | Release scheduled for November 2022 |
+| .NET 8.0 LTS | Release scheduled for November 2023 |
+
```
Comment (karelz) [inline roadmap.md]:
```suggestion
Details about longer-term schedule have been announced on 2019/5/6 in [Introducing .NET 5](https://devblogs.microsoft.com/dotnet/introducing-net-5/) blog post.

```

**Comment by kevinjpetersen** [comment]
> Thanks for your change. Let's use terminology from the article to avoid further confusion. We can add full text in parenthesis.
> We should also link the article.

Done :)

File: roadmap.md [hunk 3]
```diff
@@ -19,7 +19,13 @@ For released versions of the product:
 | Milestone                 | Release Date |
 |---------------------------|--------------|
 | .NET Core 2.2.x, 2.1.x, 1.x (servicing) | Approximately every 1-2 months or as needed (see also [releases](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md#net-core-21)) |
-| .NET Core 3.0 | [Preview releases](https://github.com/dotnet/core/blob/master/release-notes/3.0/README.md) <br/> Final version in second half of 2019 - see [Schedule in Preview 3](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-preview-3) (from 2019/3). Ship date will be announced at the Build 2019 conference. <br/> Previous announcements: [original 2018/5](https://blogs.msdn.microsoft.com/dotnet/2018/05/07/net-core-3-and-support-for-windows-desktop-applications/) and [update 2018/10](https://blogs.msdn.microsoft.com/dotnet/2018/10/04/update-on-net-core-3-0-and-net-framework-4-8/). |
+| .NET Core 3.0 | [Preview releases](https://github.com/dotnet/core/blob/master/release-notes/3.0/README.md)<br/>Release Candidate scheduled for July 2019<br/>Final Release scheduled for September 2019 |
+| .NET Core 3.1 LTS | Release scheduled for November 2019 |
```
Comment (karelz) [inline roadmap.md]:
```suggestion
| .NET Core 3.1 | LTS (Long Term Support) release, scheduled for November 2019 |
```

File: roadmap.md [hunk 4]
```diff
@@ -19,7 +19,13 @@ For released versions of the product:
...
+| .NET 5.0 | Release scheduled for November 2020 |
+| .NET 6.0 LTS | Release scheduled for November 2021 |
```
Comment (karelz) [inline roadmap.md]:
```suggestion
| .NET 6.0 | LTS (Long Term Support) release, scheduled for November 2021 |
```

File: roadmap.md [hunk 5]
```diff
@@ -19,7 +19,13 @@ For released versions of the product:
...
+| .NET 7.0 | Release scheduled for November 2022 |
+| .NET 8.0 LTS | Release scheduled for November 2023 |
```
Comment (karelz) [inline roadmap.md]:
```suggestion
| .NET 8.0 | LTS (Long Term Support) release, scheduled for November 2023 |
```

**Comment by karelz** [comment]
Sorry, our work crossed. On phone now, so it took longer ...

**Comment by kevinjpetersen** [comment]
Your changes should be added now.

File: roadmap.md [hunk 6]
```diff
@@ -19,7 +19,14 @@ For released versions of the product:
...
-| .NET Core 3.0 | [Preview releases](https://github.com/dotnet/core/blob/master/release-notes/3.0/README.md) <br/> Final version in second half of 2019 - see [Schedule in Preview 3](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-preview-3) (from 2019/3). Ship date will be announced at the Build 2019 conference. <br/> Previous announcements: [original 2018/5](https://blogs.msdn.microsoft.com/dotnet/2018/05/07/net-core-3-and-support-for-windows-desktop-applications/) and [update 2018/10](https://blogs.msdn.microsoft.com/dotnet/2018/10/04/update-on-net-core-3-0-and-net-framework-4-8/). |
+| .NET Core 3.0 | [Preview releases](https://github.com/dotnet/core/blob/master/release-notes/3.0/README.md)<br/>RC (Release Candidate) scheduled for July 2019<br/>GA (General Availability) scheduled for September 2019 |
+| .NET Core 3.1 | LTS (Long Term Support) release, scheduled for November 2019 |
+| .NET 5.0 | *For more info, see article [Introducing .NET 5](https://devblogs.microsoft.com/dotnet/introducing-net-5/).*<br/>Release scheduled for November 2020 |
```
Comment (karelz) [inline roadmap.md]: I would not make it bold + I would move the article link below the table as I suggested earlier as it refers to 5.0+ schedule

Comment (kevinjpetersen) [reply roadmap.md]: Agreed, I removed it from .NET 5.0 as I've already commited your changes under the table, and it doesn't make sense to have the link twice.

**Pull request merged**